conf/layer.conf

```diff
@@ -43,7 +36,7 @@ CMF_GIT_ROOT = "git://code.rdkcentral.com/r"
 CMF_GITHUB_MASTER_BRANCH ?= "master"
 CMF_GITHUB_ROOT ?= "git://github.com/rdkcentral"
 RDK_RELEASE ?= "1.99"
-LICENSE_FLAGS_WHITELIST += "commercial_fdk-aac"
+LICENSE_FLAGS_ACCEPTED += "commercial_fdk-aac"
```

Comment:
pls check this . we are accepting commercial.if aac not use pls remove this

Reply:
Removed this 
https://github.com/rdkcentral/meta-dac-sdk-broadband/pull/1/commits/e579dcb376b6fb910146ddc0a09baf224ef25ea4#diff-22294624f20f807485d85b0f15eac2e1dac979796d77182b94b4896eb103c243